pages/api/interest-zones/[id].ts

```diff
@@ -0,0 +1,25 @@
+import ZoneSchema from '../../../data/zone.schema';
+import { dbConnection } from '../../../services/data.service';
```

Comment:
Please rename the above file to `db-connection`, or the exported constant to DataService.